src/e2e/java/uk/gov/hmcts/reform/sscs/functional/ReminderNotificationsFunctionalTest.java

```diff
@@ -104,7 +104,6 @@ public void shouldSendNotificationsWhenAppealReceivedEventIsReceived() throws IO
     public void shouldSendNotificationsWhenDwpResponseReceivedEventIsReceived() throws IOException, NotificationClientException {
 
         triggerEvent(DWP_RESPONSE_RECEIVED_NOTIFICATION);
-        simulateCcdCallback(DWP_RESPONSE_RECEIVED_NOTIFICATION);
```

Comment:
Is this not wanted? I guess if the test passes with the same assertions it should be fine.